spark/src/test/scala/com/intenthq/pucket/spark/LocalSparkSpec.scala

```diff
@@ -0,0 +1,23 @@
+package com.intenthq.pucket.spark
+
+import org.apache.spark.{SparkConf, SparkContext}
+import org.specs2.specification.BeforeAfterAll
+
+trait LocalSparkSpec extends BeforeAfterAll with Serializable {
```

Comment:
Now we'll use `BeforeAfterAll`, a much better way of doing this sort of before/after setup in `specs2`.